src/main/js/SmeagolConfigurationForm.tsx

```diff
@@ -70,6 +71,13 @@ const SmeagolConfigurationForm: FC<Props> = ({ initialConfiguration, onConfigura
         checked={changedConfig.enabled}
         onChange={valueChangeHandler}
       />
+      <Checkbox
+        name="navLinkEnabled"
+        label={t("scm-smeagol-plugin.form.navLinkEnabled")}
+        helpText={t("scm-smeagol-plugin.form.navLinkEnabledHelp")}
+        checked={changedConfig.navLinkEnabled}
+        onChange={valueChangeHandler}
+      />
```

Comment:
The nav link option should be disabled as long as the whole configuration is not enabled.

```suggestion
      <Checkbox
        name="navLinkEnabled"
        label={t("scm-smeagol-plugin.form.navLinkEnabled")}
        helpText={t("scm-smeagol-plugin.form.navLinkEnabledHelp")}
        checked={changedConfig.navLinkEnabled}
        onChange={valueChangeHandler}
        disabled={!changedConfig.enabled}
      />
```